instadam/image.py

```diff
@@ -79,7 +79,7 @@ def get_unannotated_images():
     return jsonify({'unannotated_images': unannotated_images_res}), 200
 
 
-@bp.route('/<project_id>/image/<image_id>')
+@bp.route('/<image_id>/project/<project_id>')
```

Comment:
I guess this is a typo? Images should be resources that belong to the project.

Reply:
The blueprint has an /image prefix though, so it doesn't make sense to have an endpoint like /image/<project_id>/image/<image_id>

Reply:
> The blueprint has an /image prefix though, so it doesn't make sense to have an endpoint like /image/<project_id>/image/<image_id>

what's wrong with the original '/<project_id>/image/<image_id>'

Reply:
It makes no sense to put an endpoint '/image/<project_id>/image/<image_id>'. It just looks strange to have two /image and a <project_id> field after one of them. That'll confuse the frontend team when deciding which endpoints to make requests to.
 
This is a result of the image.py file having a url prefix of /image. Obviously it would make more sense to have 'project/<project_id>/image/<image_id>', but it's not possible in this case. Thus, the next best name is '/image/<project_id>/image/<image_id>'.